Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -331,8 +331,34 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             if did_remove:
                 pr_issue.set_labels(labels)
 
+        commits = list(repository.commits(begin=dict(hash=branch_point.hash), end=dict(branch=repository.branch)))
+
+        issue = None
+        for line in commits[0].message.split() if commits[0] and commits[0].message else []:
+            issue = Tracker.from_string(line)
+            if issue:
+                break
+
         if isinstance(remote_repo, remote.GitHub):
+            if issue and issue.redacted and args.remote is None: # TODO: is this a correct check on args.remote?
+                print("Your issue is redacted, which means it will be diverted to a secure, non-origin remote you have access to.")
+                if args.defaults or Terminal.choose(
+                    "Would you like to override this behavior? \n",
+                    default='No',
+                ) == 'No':
+                    original_remote = source_remote
+                    if len(repository.source_remotes()) < 2:
+                        print("Error. You do not have access to a secure, non-origin remote")
+                        if args.defaults or Terminal.choose(
+                            "Would you like to proceed anyways? \n",
+                            default ='No',
+                        ) == 'No':
+                            print("Failed to create pull request due to non-suitable remote")
+                            return 1
+                    source_remote = repository.source_remotes()[1]
+                    print("Switched remote from {} to {}".format(original_remote, source_remote))
```

Comment:
Would say `Making PR against '{}' instead of '{}'".format(source_remote, original_remote)`